docs/tuning-for-production.md

```diff
@@ -6,6 +6,20 @@ title: Tuning the ProxyManager for production
 
 By default, all proxy factories generate the required proxy classes at runtime.
 
+To generate proxies and store them as files, you need to use `FileWriterGeneratorStrategy` by passing it to the configuration before setting up other factories:
+
+```php
+    
+$config = new \ProxyManager\Configuration();
+
+$fileLocator = new \ProxyManager\FileLocator\FileLocator(__DIR__.'/my/generated/classes/cache/dir');
+$config->setGeneratorStrategy(new \ProxyManager\GeneratorStrategy\FileWriterGeneratorStrategy($fileLocator));
+
+// pass the configuration to proxymanager factory
+$factory = new ProxyManager\Factory\LazyLoadingValueHolderFactory($config);
+```
+This will write the generated proxies in a directory which can be used directly instead of generating them again and again.
```

Comment:
> in a directory which can be used directly instead of generating them again and again.

The `FileWriterGeneratorStrategy` writes files regardless. What we are achieving here is that the files are being written at all, which is needed for us to make them part of our deployable. Can you try adjusting the wording according to that?